openam-federation/openam-federation-library/src/main/java/jp/co/osstech/oam/saml2/plugins/PolicyCheckIDPAdapter.java

```diff
@@ -0,0 +1,339 @@
+/**
+ * DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS HEADER.
+ *
+ * Copyright (c) 2019 Open Source Solution Technology Corporation
+ * All Rights Reserved.
+ *
+ * The contents of this file are subject to the terms
+ * of the Common Development and Distribution License
+ * (the License). You may not use this file except in
+ * compliance with the License.
+ *
+ * You can obtain a copy of the License at
+ * http://forgerock.org/license/CDDLv1.0.html 
+ * See the License for the specific language governing
+ * permission and limitations under the License.
+ *
+ * When distributing Covered Code, include this CDDL
+ * Header Notice in each file and include the License file
+ * at http://forgerock.org/license/CDDLv1.0.html 
+ * If applicable, add the following below the CDDL Header,
+ * with the fields enclosed by brackets [] replaced by
+ * your own identifying information:
+ * "Portions Copyrighted [year] [name of copyright owner]"
+ *
+ */
+
+package jp.co.osstech.oam.saml2.plugins;
+
+import com.iplanet.sso.SSOToken;
+import com.sun.identity.authentication.util.AMAuthUtils;
+import com.sun.identity.authentication.util.ISAuthConstants;
+import com.sun.identity.entitlement.Entitlement;
+import com.sun.identity.entitlement.EntitlementException;
+import com.sun.identity.entitlement.Evaluator;
+import com.sun.identity.entitlement.opensso.SubjectUtils;
+import com.sun.identity.federation.common.FSUtils;
+import com.sun.identity.policy.interfaces.Condition;
```

Comment:
`com.sun.identity.policy.interfaces.Condition` is a deprecated class. How about replacing with `org.forgerock.openam.entitlement.conditions.environment.ConditionConstants` ?

Reply:
I agree.